css/style.css

```
@@ -849,54 +1240,295 @@ img {
+  flex-wrap: wrap;
+  align-items: center;
   width: 425px;
+  padding-top: 9px;
+  padding-right: 14px;
+  padding-bottom: 9px;
+  padding-left: 14px;
 }
 
 .form-fat label {
   line-height: 18px;
   font-weight: 500;
 }
 
+.form-radio[type="radio"] {
+  display: none;
+}
+
+.filter-fat-field {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 19px;
+}
+
+.filter-fat-row svg:nth-of-type(2) {
+  position: absolute;
+  top: 7px;
+  left: 7px;
+}
+
+.filter-fat-field:last-child {
+  margin-right: 0;
+}
+
+.filter-fat-field label {
+  margin-left: 11px;
+  padding-left: 22px;
+  position: relative;
+  display: block;
+}
+
+.filter-fat-field label::before {
+  position: absolute;
+  top: -2px;
+  left: -11px;
+  content: "";
+  width: 22px;
+  height: 22px;
+  background-image: url("../img/radio.svg");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  opacity: 0.8;
+}
+
+.form-radio:checked+label::after {
+  position: absolute;
+  top: 5px;
+  left: -4px;
+  content: "";
+  width: 8px;
+  height: 8px;
+  background-image: url("../img/radio-on.svg");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  opacity: 0.8;
+}
+
+.form-radio:hover+label::before,
+.form-radio:hover+label::after,
+.form-radio-label:hover::before,
+.form-radio-label:hover::after,
+.form-radio:focus+label::before,
+.form-radio:focus+label::after,
+.form-radio-label:focus::before,
+.form-radio-label:focus::after,
+.form-radio:active+label::before,
+.form-radio:active+label::after,
+.form-radio-label:active::before,
+.form-radio-label:active::after {
+  opacity: 1;
+}
+
+.form-radio:disabled+label::before,
+.form-radio:disabled+label::after {
+  opacity: 0.4;
+}
+
 .filter-row {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 9px;
+  padding-right: 16px;
+  padding-bottom: 9px;
+  padding-left: 16px;
   width: 711px;
   line-height: 18px;
   font-weight: 500;
 }
 
-.filter-fat-field,
+.filter-checkbox-input {
+  display: none;
+}
+
+.filter-checkbox-label {
+  position: relative;
+  padding-left: 32px;
+}
+
+.filter-field:nth-of-type(3) {
+  margin-right: 25px;
+}
+
+.filter-field:nth-of-type(3) .filter-checkbox-label {
+  padding-left: 35px;
+}
+
+.filter-field:nth-of-type(4) .filter-checkbox-label {
+  padding-left: 33px;
+}
+
+.filter-field:nth-of-type(5) .filter-checkbox-label {
+  padding-left: 33px;
+}
+
+.filter-checkbox-input+.filter-checkbox-label::before {
+  position: absolute;
+  top: 0;
+  left: 0;
+  content: "";
+  width: 20px;
+  height: 20px;
+  background-image: url("../img/checkbox-off.svg");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  opacity: 0.8;
+}
+
+.filter-checkbox-input:checked+.filter-checkbox-label::before {
+  position: absolute;
+  top: 0;
+  left: 0;
+  content: "";
+  width: 24px;
+  height: 20px;
+  background-image: url("../img/checkbox-on.svg");
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  opacity: 0.8;
```

Здесь меняются только 2 свойства - их и стоит переопределить
```suggestion
  width: 24px;
  background-image: url("../img/checkbox-on.svg");
```